playwright/tests/pageTypes/pageTypesTable.spec.ts

```diff
@@ -120,3 +120,35 @@ test.run()(
     await expect(pageTypesTablePage.getRow(newPageTypeName)).toBeVisible();
   },
 );
+
+test.run()(
+  "I can delete a page type",
+  async ({ pageTypesTablePage, reusablePageType }) => {
+    await pageTypesTablePage.goto();
+    await pageTypesTablePage.openActionMenu(reusablePageType.name, "Remove");
+
+    await pageTypesTablePage.deleteTypeDialog.deleteType();
+
+    await expect(
+      pageTypesTablePage.getRow(reusablePageType.name),
+    ).not.toBeVisible();
+    await pageTypesTablePage.page.reload();
+    await expect(
+      pageTypesTablePage.getRow(reusablePageType.name),
+    ).not.toBeVisible();
+  },
+);
+
+test.run()(
+  "I can see the blank slate message when I don't have any custom types",
+  async ({ pageTypesTablePage, procedures }) => {
+    procedures.mock(
+      "customTypes.readAllCustomTypes",
+      () => ({ models: [], errors: [] }),
+      { execute: false },
+    );
+    await pageTypesTablePage.goto();
+
+    await expect(pageTypesTablePage.blankSlate).toBeVisible();
```

Comment:
Test don't pass 👀.

But I was thinking to test closer to the user without testId, maybe we should more verify the blank slate description and button? With that no need to a testId for the blankSlate, WDYT?

We could also check that clicking on the button open the create modal so we ensure user is not blocked on the blank slate forever. WDYT?

Reply:
I'm never sure if we really need to locate the item first. It's more if the text can be at two different places why not. But here for example it seems the description and create button can only be there within the blank slate. It's even more powerful with the id and I don't have a strong opinion so LGTM 🙂

Reply:
Btw relative to the todo you wrote. I think I may have fixed the problem so you may not need that overhead and the redirection should be perfect! Thanks to the removal of the saga, I forgot about these todo! 👀 Could you check if there are required and remove them if not?

Reply:
The description is probably fine to check if it is present anywhere on the page, but the "Create" button could be confused with the non-blank-slate "Create" button. We don't currently show two Create buttons, but I wonder if we should be explicit about which button we are checking for in case we have a regression somewhere.

Regarding the TODO: You are totally right! I tested it without the extra navigation before, and it failed. But now it works without it. There must have been something else wrong with the test back then.

I'll remove the other uses `goto()`.

Reply:
In this case I guess it's nice to be specific yes as we know we have another button with the same label, so all good with the testid 👍

Super cool! Something less to take care haha